scripts/communications/notification_manager.py

```
@@ -0,0 +1,1477 @@
+Unified notification system for change-process automation.
+Supports multiple notification channels with consistent interface.
+"""
+
+import logging
+import smtplib
+import requests
+import re
+from email.mime.text import MIMEText
+from email.mime.multipart import MIMEMultipart
+from email.mime.base import MIMEBase
+from email.mime.application import MIMEApplication
```

Ruff (0.11.9)

`email.mime.application.MIMEApplication` imported but unused

Remove unused import: `email.mime.application.MIMEApplication`

(F401)